src/Parameters/Config/DocumentOptions.php

```diff
@@ -20,25 +20,32 @@
 
 namespace BigBlueButton\Parameters\Config;
 
-class DocumentOptionsStore
-{
-    private $attributes = [];
+use BigBlueButton\Enum\DocumentOption;
 
-    public function __construct(array $attributes = [])
-    {
-        $this->attributes = $attributes;
-    }
+class DocumentOptions
+{
+    /** @var array<string, bool> */
+    private array $documentOptions = [];
 
-    public function addAttribute($name, $value)
+    /**
+     * @throws \Exception
+     */
+    public function addOption(string $documentOption, bool $value): self
     {
-        if (is_bool($value)) {
-            $value = $value ? 'true' : 'false';
+        if (!DocumentOption::hasValue($documentOption)) {
```

Comment:
I would avoid raising Exceptions, always thinking of custom BigBlueButton builds, dozens of them are in the wild.

Reply:
Exception removed by using ENUM-Type class of php81